third_party/mdbook/book.js

```diff
@@ -69,7 +69,7 @@ function playground_text(playground, hidden = true) {
                     name: "run",
                     bindKey: {
                         win: "Ctrl-Enter",
-                        mac: "Ctrl-Enter"
+                        mac: "Cmd-Enter"
```

Comment:
"Cmd-Enter" is more convenient on my machine, but I can revert this change, if it is only me :)

Reply:
I'm on Linux, so I wouldn't know -- but please make this change in it's own PR so that we can discuss it there :smile: I would suggest pinging the other instructors on the PR to give them a chance to chime in.

Reply:
Reverted this change.